CHANGELOG.md

```diff
@@ -194,16 +210,16 @@ UPGRADE:
 OVERVIEW:
 
 Release 1.8.0 includes:
-* Support for NGINX App Protect Web Application Firewall. 
```

Comment:
I feel we need to go through all files and remove trailing spaces instead of doing that in each PR 😄

Reply:
yeah VS Code does it every time I touch a file, need to find out if it's possible to do it everywhere 😄